source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -150,62 +150,41 @@ Configure and run CRI-O + kata-runtime
 
        sudo systemctl daemon-reload
        sudo systemctl restart crio
-       sudo systemctl restart kubelet
 
 #.  Initialize the master control plane with the command:
 
     .. code-block:: bash
 
        sudo kubeadm init --cri-socket=/run/crio/crio.sock
 
-
 Install pod network add-on
 **************************
 
 You must choose and install a `pod network add-on`_ to allow your pods to
 communicate. Check whether or not your add-on requires special flags when you
 initialize the master control plane.
 
-**Notes about flannel add-on**
-
-If you choose the `flannel` add-on, then you must add the following to the
-`kubeadm init` command:
-
-..  code-block:: bash
-
-    --pod-network-cidr 10.244.0.0/16
-
-If you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
-
-..  code-block:: bash
-
-    [crio.runtime]
-    manage_network_ns_lifecycle = true
-
-Create a symlink for the network overlays:
-
-.. code-block:: bash
-
-   sudo ln -s /usr/libexec/cni /opt/cni/bin
+The CRI-O default plugin_dir is :file:`/opt/cni/bin`. This must be a
+writable directory because third-party networking add-ons will install
+themselves there.
 
 .. note::
 
-   |CL| installs CNI plugins that are part of the `cloud-native-basic`
-   bundle to :file:`/usr/libexec/cni`. The directory is required because `
-   swupd verify` may use it to repair a system to a known good state.
-
-**Notes about Weave Net add-on**
+   CNI plugins provided by |CL| are installed as part of *cloud-native-basic*
+   in :file:`/usr/libexec/cni/` and are currently *not* found by CRI-O by
+   default. These separate directories are required because `swupd` controls
+   the content of :file:`/usr` and leaves :file:`/opt` unchanged.
 
-If you choose the `Weave Net` add-on, you must make the following
-changes because it installs itself in the :file:`/opt/cni/bin` directory.
+When using third-party network add-ons that rely on those plugins, such as
+Weave or Flannel do, make them available by creating symlinks:
 
```

Comment:
For Weave, no specific steps are needed besides the generic setup of /opt/cni/bin. Perhaps it's worth mentioning that? There's a difference between "we don't have instructions for xyz" and "xyz just works".

Reply:
Review https://github.com/clearlinux/clear-linux-documentation/pull/393/commits/f67cf636d71687b0d02d1a371a6049af440021ea